.dockerignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Should this file be added?

Reply:
Speeds up build time with docker, as we don't need anything in docker context.